src/ubuntu/22.04/cross/arm64/Dockerfile

```diff
@@ -4,6 +4,13 @@ FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-crossdeps-local
 RUN apt-get update \
     && apt-get install -y \
         binutils-aarch64-linux-gnu \
+        nodejs \
     && rm -rf /var/lib/apt/lists/*
 
 ADD rootfs.arm64.tar crossrootfs
+
+# update node
+ENV NODE_VERSION 20.9.0
+RUN cd ~ && \
+    curl -LO https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-arm64.tar.xz && \
+    tar xf node-v${NODE_VERSION}-linux-arm64.tar.xz
```

Comment:
I'm trying to replace this locally. The directory gets extracted into `~` but how is it then automatically used during the (source) build?

Reply:
These steps worked for me: https://github.com/dotnet/source-build/issues/3924#issuecomment-1886811967

So instead of doing a `sudo apt-get install node` above, you would just do:
```bash
curl -sL https://deb.nodesource.com/setup_20.x -o nodesource_setup.sh
sudo bash nodesource_setup.sh
sudo apt-get install node -y
```